src/index.html

```diff
@@ -6,13 +6,119 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Calendar</title>
+
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="wrapper">
```

Comment:
@mrkovaliv I don't know how to make it real. 

we need the calendar to be positioned in the center.
I can't use absolute position because there are several calendars. 
I can make left margin with help of "margin: 0 auto". 
But what should I do with top margin?